src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -38,14 +41,25 @@ class Bic extends Constraint
     );
 
     public $message = 'This is not a valid Business Identifier Code (BIC).';
+    public $ibanMessage = 'This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.';
+    public $iban;
+    public $ibanPropertyPath;
 
-    public function __construct($options = null)
+    public function __construct(array $options = null)
```

Comment:
not sure about this :)

Reply:
looks good for me as we expect an array.

Why aren't you sure about this?

Reply:
In this case, the change looks valid to me. Generally you can define a default option by overriding the `getDefaultOption()` method. But since that is not the case for the `Bic` constraint, it will already throw an exception when no array is passed.

Reply:
exactly, but now we'll never reach the exception at https://github.com/symfony/symfony/blob/491bbe741dc0762eeda59ceb7d441a520af8d878/src/Symfony/Component/Validator/Constraint.php#L136


so basically all constraints will throw this exception, except here it will cause a type error: https://3v4l.org/WjMuY